lib/SIL/AbstractionPattern.cpp

```diff
@@ -78,6 +78,8 @@ TypeConverter::getAbstractionPattern(VarDecl *var, bool isNonObjC) {
     swiftType = getLoweredBridgedType(
         AbstractionPattern(genericSig, swiftType, clangType),
         contextType,
+        isa<clang::ObjCPropertyDecl>(clangDecl)
+          ? Bridgeability::Full : Bridgeability::None,
```

Comment:
This doesn't seem sufficient, since we can also import functions as getters using import-as-member.

Reply:
Good point.  I should probably check specifically for `!isa<VarDecl> && !isa<FieldDecl>`.

Reply:
Don't forget IndirectFieldDecl.